packages/cashier/src/modules/deposit-crypto/components/deposit-crypto-side-notes/deposit-crypto-side-notes.tsx

```diff
@@ -1,20 +1,8 @@
 import React from 'react';
-import { useCurrentCurrencyConfig } from '@deriv/hooks';
-import { observer } from '@deriv/stores';
 import { CryptoTransactionsSideNoteRecentTransaction } from '../../../crypto-transactions/components';
-import DepositCryptoSideNoteUSDT from './deposit-crypto-side-note-usdt';
-import './deposit-crypto-side-notes.scss';
 
-const DepositCryptoSideNotes: React.FC = observer(() => {
-    const currency_config = useCurrentCurrencyConfig();
-
-    return (
-        <>
-            <CryptoTransactionsSideNoteRecentTransaction />
-            {currency_config.is_USDT && <DepositCryptoSideNoteUSDT currency='USDT' />}
-            {currency_config.is_eUSDT && <DepositCryptoSideNoteUSDT currency='eUSDT' />}
-        </>
-    );
-});
+const DepositCryptoSideNotes: React.FC = () => {
+    return <CryptoTransactionsSideNoteRecentTransaction />;
+};
```

Comment:
here i decided to leave `DepositCryptoSideNotes` component in case if we will add new crypto depsoit side-notes. So it's better to have DepositCryptoSideNotes container for future side-notes.
@farzin-deriv